src/gf.c

```diff
@@ -11,13 +11,11 @@
 #include <stdlib.h>
 #include <string.h>
 #include <assert.h>
-#ifdef _OS_WINDOWS_
-#include <malloc.h>
```

Comment:
Yeah but only since https://github.com/JuliaLang/julia/commit/145687fb11d43a8494e610e2fafab8109696c50f#diff-53dc66d2cb199a2ea107533c1f99f57b. It's fine on mingw-w64 but not on msvc, apparently.